examples/time_frequency/source_power_spectrum_opm.py

```diff
@@ -8,7 +8,8 @@
 Here we compute the resting state from raw for data recorded using
 a Neuromag VectorView system and a custom OPM system.
 The pipeline is meant to mostly follow the Brainstorm :footcite:`TadelEtAl2011`
-`OMEGA resting tutorial pipeline <bst_omega_>`_.
+`OMEGA resting tutorial pipeline
+<https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega>`__.
```

Comment:
Too many underscores?

Reply:
That was intentional, it's an anonymous hyperlink in RST (which IIUC is for when you don't plan to reuse the same link name later)